docs/reference/settings/health-api-settings.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[health-api-settings]]
+=== Health API settings in {es}
+[subs="attributes"]
+++++
+<titleabbrev>Health API settings</titleabbrev>
+++++
+
+The following are the _expert-level_ settings available for configuring the
+<<health-api, Health API>>. It is not recommended to change any of these
+from their default values.
+
+==== Cluster level settings
+
+`health.master_history.has_master_lookup_timeframe`::
+(<<static-cluster-setting,Static>>) The amount of time the API looks back to see if the cluster has had
```

Comment:
I'd say we're looking back to see if an instance observed any masters. We're not making cluster-level decisions here.

What do you think?